src/quo2/components/drawers/permission_context/style.cljs

```diff
@@ -3,9 +3,13 @@
 
 (def radius 20)
 (def container
-  {:padding-top             16
+  {:flex-direction          :row
```

Comment:
this component was missing some properties needed it for it to match designs easier